services-api/src/main/java/io/scalecube/services/Reflect.java

```diff
@@ -5,16 +5,12 @@
 import static io.scalecube.services.CommunicationMode.REQUEST_RESPONSE;
```

Comment:
Please  don't  re-arrange functions. It's hard to review what was changed. Please revert back methods aligment as it was.